extensions/kubectx/src/lib/cli.ts

```diff
@@ -0,0 +1,14 @@
+import { accessSync, constants } from "fs";
+import { cpus } from "os";
+
+export const getBrewExecutablePath = (packageName: string) => {
```

Comment:
@pernielsentikaer 

I think preferences form is not best **(easiest for user)** way because i as a user i don't want to search brew installed path. 

https://developers.raycast.com/api-reference/preferences#preferences

package.json's preferences section allows **default value** but package.json is not programmable, we should show different paths for M1 and Intel.

I prefer this solution:

```ts
export const getBrewExecutablePath = (packageName: string) => {
  const possiblePaths = [`/usr/local/bin/${packageName}`, `/opt/homebrew/bin/${packageName}`];

  let validPath = packageName;

  possiblePaths.some((path) => {
    try {
      accessSync(path, constants.X_OK);

      validPath = path;

      return true;
    } catch (e) {
      return false;
    }
  });

  return validPath;
};
```

Reply:
That could indeed work, but sometimes it can be located another place, you can not 100% say where the user have brew installed afaik. See other plugins doing [like this](https://github.com/raycast/extensions/blob/main/extensions/brew-services/src/services.tsx#L23)

If you leave package.json empty, then it's only if the user add path is used, and then you can do your check as above?